src/WebJobs.Script/ManagedDependencies/ManagedDependencyConfigs.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    internal class ManagedDependencyConfigs : IManagedDependencyConfigs
+    {
+        public string RapidUpdateAzModulePath { get => @"D:\Program Files (x86)\ManagedDependencies\PowerShell\AzPSModules"; }
```

Comment:
`D:\Program Files (x86)` can be resolved using `Environment.ExpandEnvironmentVariables(ScriptConstants.ProgramFilesX86`

Reply:
Changed ManagedDependencyProvider  to use IEnvironment